tap_gitlab/__init__.py

```diff
@@ -55,7 +59,7 @@ def load_schema(entity):
         'replication_method': 'FULL_TABLE',
     },
     'commits': {
-        'url': '/projects/{id}/repository/commits?since={start_date}&with_stats=true',
```

Comment:
@ninomllr I'm wondering if it might be better to make this an optional config which is defaulted to false. I'm just hesitant to potentially add significantly more data to existing pipelines, especially those with large repo/projects.

Maybe in [sync_commits](https://github.com/MeltanoLabs/tap-gitlab/blob/f5449f80321c95ef717a05480e505b1c88c2448b/tap_gitlab/__init__.py#L327 ) we just append that to the `all=true` parameter if a config setting like `fetch_all_commits` is set to true.

What do you think?

Reply:
This is actually the only thing that I changed. The rest was my IDE that reformatted everything automatically, sorry.
I will change this as soon as I get to it to make it optional.

Reply:
Would it be ok if I leave the new formatting? I think it makes sense, even if it was hard to read.

Reply:
I agree, since this could be considered a breaking change for some existing projects, it would be good to set as an optional config setting - like `fetch_all_commits` (boolean defaulting to `false`) or similar, as @pnadolny13 suggests.

Reply:
In regards to the formatting, I'll leave that to @pnadolny13. I'm fine with either direction, and it's helpful that you've called out the changed lines. Thanks.

Reply:
@ninomllr Awesome, thanks! Just mention me/AJ whenever youre ready for a re-review. For the formatting changes, its fine as long as its clear what is actually being changed so we can do a proper review.

Reply:
I will anyway redo my change, because for some reason my IDE did mess up quite a lot of files. Will ping you when it's ready.